db/fixtures/miq_product_features.yml

```diff
@@ -5129,6 +5137,10 @@
         :description: Suspend VM
         :feature_type: control
         :identifier: vm_suspend
+      - :name: Pause
+        :description: Pause VM
+        :feature_type: control
+        :identifier: vm_pause
```

Comment:
The ui only exposes "suspend" action. While some of the infra providers may support pause action, it is not available through webui atm.